packages/core/src/browser/core-preferences.ts

```diff
@@ -136,6 +136,20 @@ export const corePreferenceSchema: PreferenceSchema = {
             default: isOSX ? 1500 : 500,
             description: nls.localizeByDefault('Controls the delay in milliseconds after which the hover is shown.')
         },
+        'workbench.sash.hoverDelay': {
+            type: 'number',
+            default: 1,
+            minimum: 1,
+            maximum: 2000,
+            description: nls.localizeByDefault('Controls the hover feedback delay in milliseconds of the dragging area in between views/editors.')
+        },
+        'workbench.sash.size': {
+            type: 'number',
+            default: 7,
+            minimum: 1,
+            maximum: 20,
+            description: nls.localizeByDefault('Controls the feedback area size in pixels of the dragging area in between views/editors. Set it to a larger value if needed.')
+        },
```

Comment:
Any reason we do not want to use the same values and defaults as in vscode?

```ts
'workbench.sash.size': {
	type: 'number',
	default: isIOS ? 20 : 4,
	minimum: 1,
	maximum: 20,
	description: localize('sashSize', "Controls the feedback area size in pixels of the dragging area in between views/editors. Set it to a larger value if you feel it's hard to resize views using the mouse.")
},
'workbench.sash.hoverDelay': {
	type: 'number',
	default: 300,
	minimum: 0,
	maximum: 2000,
	description: localize('sashHoverDelay', "Controls the hover feedback delay in milliseconds of the dragging area in between views/editors.")
},
```

Reply:
@vince-fugnitto I have updated the values and defaults to align with the behavior present in vscode.

I kept the default value for `sash.hoverDelay` to be 0 because it solves resizing latency issue mentioned in #10416.